core/src/com/fourinachamber/fortyfive/keyInput/KeyInputMap.kt

```diff
@@ -76,7 +85,9 @@ class KeyInputMap(
                         }
                     }
             }
-        return bestCandidate?.invoke(screen, keycode) ?: false
+        var toReturn = false
+        bestCandidate?.forEach { if (it.invoke(screen, keycode)) toReturn = true }
```

Comment:
this will very likely BREAK on merge, as you didn't merge rework into your branch apparently, so fix this beforehand